common/lib/xmodule/xmodule/course_module.py

```diff
@@ -846,6 +846,15 @@ class CourseFields(object):
         scope=Scope.settings,
     )
 
+    teams_configuration = Dict(
+        display_name=_("Teams Configuration"),
+        help=_(
+            "Enter policy keys and values to enable the teams feature, define topics, "
+            "and specify the maximum team size."
```

Comment:
The help text is not very good right now, I agree. There's an example of the structure in the ticket: https://openedx.atlassian.net/browse/TNL-1889